### PR TITLE
added annotations to subsription pojo

### PR DESCRIPTION
**Pull request by glozy**

### CODE REVIEW

File: src/main/java/com/revature/pojos/Subscription.java [hunk 1]
```diff
@@ -1,44 +1,66 @@
...
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="SUBSCRIPTION")
```
Comment (devwithdonald) [inline src/main/java/com/revature/pojos/Subscription.java]: table name is wrong, should be "SUBSCRIPTIONS"

File: src/main/java/com/revature/pojos/BasicUser.java [hunk 2]
```diff
@@ -2,9 +2,21 @@
...
+@Table(name="BASIC_USER")
 public class BasicUser extends User{
+	
+	@Column(name="FIRST_NAME")
 	private String firstName;
+	
+	@Column(name="LAST_NAME")
 	private String lastName;
+	
+
```
Comment (devwithdonald) [inline src/main/java/com/revature/pojos/BasicUser.java]: can you put a // TODO here so we know we gotta go back to it

File: src/main/java/com/revature/pojos/Subscription.java [hunk 3]
```diff
@@ -1,44 +1,66 @@
...
-	private int subscriptionID;
+	@Id
+	@Column(name="SUBSCRIPTION_ID")
+	@GeneratedValue(strategy=GenerationType.IDENTITY)
+	private int subscriptionId;
 	
-	private int userID;
+	@ManyToOne(cascade=CascadeType.ALL, fetch=FetchType.LAZY)
+	@JoinColumn(name="USER_ID")
+	private int userId;
```
Comment (devwithdonald) [inline src/main/java/com/revature/pojos/Subscription.java]: i think this needs to be - private User user;

File: src/main/java/com/revature/pojos/Subscription.java [hunk 4]
```diff
@@ -1,44 +1,66 @@
...
 	
-	private int businessID;
+	@ManyToOne(cascade=CascadeType.ALL, fetch=FetchType.LAZY)
+	@JoinColumn(name="BUSINESS_ID")
+	private int businessId;
```
Comment (devwithdonald) [inline src/main/java/com/revature/pojos/Subscription.java]: private BusinessUser businessUser;

File: src/main/java/com/revature/pojos/Subscription.java (hunk 1 above)
Comment (glozy) [reply src/main/java/com/revature/pojos/Subscription.java]: okk, missed that

File: src/main/java/com/revature/pojos/BasicUser.java (hunk 2 above)
Comment (glozy) [reply src/main/java/com/revature/pojos/BasicUser.java]: sure

File: src/main/java/com/revature/pojos/Subscription.java (hunk 3 above)
Comment (glozy) [reply src/main/java/com/revature/pojos/Subscription.java]: okk

File: src/main/java/com/revature/pojos/Subscription.java (hunk 4 above)
Comment (glozy) [reply src/main/java/com/revature/pojos/Subscription.java]: alright

**Pull request closed without merging**